bellbot_gui/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>bellbot_gui</name>
-  <version>0.0.10</version>
+  <version>0.0.11</version>
```

Comment:
A ok, I thought you meant by PR that I had to do this... Didn't know these are changed by jenkins... How to fix that? Shall I manually change it back to 0.0.10 and commit?